src/app/component-library/catalog/asset-detail-dialog/asset-detail-dialog.component.html

```diff
@@ -97,7 +97,8 @@
       <button
         *ngIf="
           data.type === 'contract-agreement' &&
-          data.contractAgreement?.direction === 'CONSUMING'
+          data.contractAgreement?.direction === 'CONSUMING' &&
+          data.contractAgreement?.statusText === 'Active'
```

Comment:
if we really need to check for this here, we should add another boolean / enum field "status: 'ACTIVE' | 'DISABLED'" / "canTransfer: boolean" or sth. 

Also this button should not disappear. We want to be as transparent as possible to what is happening, why it's happening. A possible solution is, that it should be:
- Disabled
- With an info icon
- With a tooltip